catalog.html

```diff
@@ -110,188 +112,215 @@ <h2 class="visually-hidden">Форма поиска отелей</h2>
+                <option value="" selected>Сначала дешевые</option>
+                <option value="dog">Сначала дорогие</option>
+              </select>
 
-          <a class="buttin primary" href="#">подробнее</a>
-          <button class="buttin button-favorites">в избранное</button>
+              <div class="switch-card">
+                <button class="button-switch">
+                  <span class="visually-hidden">Показать карточки</span>
+                </button>
 
-          <p class="stars-4">
-            <span class="visually-hidden">4 звезды</span>
-          </p>
-          <p class="hotel-card-rating">Рейтинг: 8,5</p>
-        </li>
+                <button class="button-switch">
+                  <span class="visually-hidden">Показать большими карточками</span>
+                </button>
 
-        <li class="hotel-card-item">
-          <a class="hotel-card-link" href="#" tabindex="-1">
-          <img class="hotel-card-img" src="images/photo-hotels/photo-2.jpg" width="300" height="212" alt="Гостиница: Villas at Poco Diablo">
-          </a>
+                <button class="button-switch">
+                  <span class="visually-hidden">Показать список</span>
+                </button>
+              </div>
+            </div>
 
-          <a class="hotel-card-link-title" href="#" tabindex="-1">
-            <h3 class="hotel-card-title">Villas at Poco Diablo</h3>
-          </a>
+          </div>
 
-          <p class="hotel-card-name">Гостиница</p>
-          <p class="hotel-card-price">От 5000 ₽</p>
+          <div class="hotel-card-wrap">
+            <ul class="hotel-card-list">
 
-          <a class="buttin primary" href="#">подробнее</a>
-          <button class="buttin button-in-favorites">в избранном</button>
+              <li class="hotel-card-item">
+                <a class="hotel-card-link" href="#" tabindex="-1">
+                  <img class="hotel-card-img" src="images/photo-hotels/photo-1.jpg" width="300" height="212" alt="Гостиница: Amara Resort &amp; Spa">
+                </a>
 
-          <p class="stars-4">
-            <span class="visually-hidden">4 звезды</span>
-          </p>
-          <p class="hotel-rating">Рейтинг: 9,2</p>
-        </li>
+                <a class="hotel-card-link-title" href="#" tabindex="-1">
+                  <h3 class="hotel-card-title">Amara Resort &amp; Spa</h3>
+                </a>
+
+                <div class="hotel-card-content">
+                  <p class="hotel-card-name">Гостиница</p>
+                  <p class="hotel-card-price">От 4000 ₽</p>
+
+                  <a class="button button-primary" href="#">подробнее</a>
+                  <button class="button button-favorites">в избранное</button>
+
+                  <p class="stars-4">
+                    <span class="visually-hidden">4 звезды</span>
+                  </p>
+                  <p class="hotel-card-rating">Рейтинг: 8,5</p>
+                </div>
```

Comment:
```suggestion
                  <p class="hotel-card-name">Гостиница</p>
                  <p class="hotel-card-price">От 4000 ₽</p>

                  <a class="button button-primary" href="#">подробнее</a>
                  <button class="button button-favorites">в избранное</button>

                  <p class="stars-4">
                    <span class="visually-hidden">4 звезды</span>
                  </p>
                  <p class="hotel-card-rating">Рейтинг: 8,5</p>
```

hotel-card-content - лишний, зачем он?